app/views/layouts/_header.html.erb

```diff
@@ -68,6 +69,14 @@
 </div>
 
 <script type="text/javascript">
+
+document.addEventListener("DOMContentLoaded", function () {
+  var gettingStartedLink = document.querySelector("#navbar-dropdown-1, #navbar-dropdown-2");
```

Comment:
In the commited code, there is no element with id `navbar-dropdown-2`. In query selector is it required ?